app/views/bulkrax/entries/show.html.erb

```diff
@@ -36,41 +37,46 @@
       <% end %>
     </p>
 
+    <%# Consider spliting dynamic fields into two partials %>
+
     <p>
       <% if @entry.last_error.present? %>
         <strong>Errored at:</strong> <%= @entry.last_error_at %><br />
         <strong>Error:</strong> <%= @entry.last_error['error_class'] %> - <%= @entry.last_error['error_message'] %><br />
         <strong>Error Trace: </strong> <button id="entry_error">Toggle</button>
         <p id="error_trace" style="display: none"><%= @entry.last_error['error_trace'] %></p>
-      <% else %>
+      <% elsif @entry.last_succeeded_at != nil %>
         <strong>Succeeded At:</strong> <%= @entry.last_succeeded_at %>
+      <% else %>
+        <strong>Succeeded At:</strong> Item has not been <%= @importer.present? ? 'imported' : 'exported' %> successfully
       <% end %>
     </p>
 
     <p>
-      <% if @entry.last_succeeded_at != nil %>
-        <% if @importer.class == Bulkrax::Importer %>
-          <strong>Importer:</strong>
-          <%= link_to @importer.name, importer_path(@importer) %>
-        <% else %>
-        <%# TODO: Create Exporter showpage table for individual items in order to link to the item %>
-          <strong>Exporter Link: Showpage has not been setup yet.</strong>
-        <% end %>
-      <% else %>
-        <strong>Importer Link: Item has not been imported successfully</strong>
+      <% if @importer.present? %>
+        <strong>Importer:</strong>
+        <%= link_to @importer.name, importer_path(@importer) %>
+      <% elsif @exporter.present? %>
+        <strong>Exporter:</strong>
+        <%= link_to @exporter.name, exporter_path(@exporter) %>
       <% end %>
     </p>
 
-    <p>
-      <% factory_record = @entry.factory.find %>
-      <% if @entry.last_succeeded_at != nil && factory_record.present? %>
-        <strong><%= @entry.factory_class.to_s %> Link:</strong>
-        <% if @entry.factory_class.to_s == "Collection" %>
-          <%= link_to @entry.factory_class.to_s, hyrax.polymorphic_path(@entry.factory.find) %>
+    <%# TODO: Currently no Exporter equivalent, is one necessary? %>
+    <% if @importer.present? %>
+      <p>
+        <% factory_record = @entry.factory.find %>
+        <% if @entry.last_succeeded_at != nil && factory_record.present? %>
+          <strong><%= @entry.factory_class.to_s %> Link:</strong>
+          <% if @entry.factory_class.to_s == "Collection" %>
+            <%= link_to @entry.factory_class.to_s, hyrax.polymorphic_path(@entry.factory.find) %>
+          <% else %>
+            <%= link_to @entry.factory_class.to_s, main_app.polymorphic_path(@entry.factory.find) %>
+          <% end %>
```

Comment:
**Question** 

Is this link (see here: https://share.getcloudapp.com/RBudbgNP) necessary on the Exporter show page? 

The current `Bulkrax::ObjectFactory` class only supports `Importers`, refactoring to that class would be necessary to get this to work for `Exporters`. Also, there is a custom `Bulkrax::OregonDigitalObjectFactory` override class in OD that would also need to be refactored.

Reply:
I think linking to the actual item is pretty cool for closing the loop, I don't see anything that really prevents that code from being moved back in to the exporter.  The factory itself doesn't change, just the code at the bottom of Importerbehavior.  @geekscruff what do you think?

Reply:
The problem I was seeing was that [this line at the bottom of ImporterBehavior](https://github.com/samvera-labs/bulkrax/blob/master/app/models/concerns/bulkrax/import_behavior.rb#L62) passes `#replace_files` as a param, which is undefined on Exporter. I tried a couple fixes (if it's an exporter, pass false instead of `#replace_files`, etc) when I realized there's an override `#factory` method that would also need to be changed [here](https://gitlab.com/notch8/oregon-digital/-/blob/bulkrax/app/models/concerns/bulkrax/has_local_processing.rb#L6). 

All that being said, what would the best way to fix this be?

Reply:
Given the exporter is exporting a work, doesn't each entry know what that work is without needing to refer to the factory (which it shouldn't have to as it isn't creating/updating works)?

Reply:
there's a method in exporter_bahaviour.rb called work - wouldn't that do it?

Reply:
Thanks @geekscruff! https://github.com/samvera-labs/bulkrax/pull/177/commits/3e1c33d35cb7a63ced426beabc3825c09e91ae0f